src/diffusers/loaders.py

```diff
@@ -112,14 +116,14 @@ def _fuse_lora(self):
     def _unfuse_lora(self):
         if not (hasattr(self, "w_up") and hasattr(self, "w_down")):
             return
-        logger.info(f"Unfusing LoRA weights for {self.__class__}")
```

Comment:
The problem with "info" here is that it would pop up 100+ times for models like SDXL which makes it unreadable. Think we should better remove it.